init-letsencrypt.sh

```diff
@@ -6,6 +6,11 @@ data_path="./data/certbot"
 email="" # Adding a valid address is strongly recommended
 staging=0 # Set to 1 if you're testing your setup to avoid hitting request limits
 
+if [ "$EUID" -ne 0 ]; then
```

Comment:
Why should this be run as root? I don’t think the script should require root privileges.

Reply:
I made root privileges optional.